### PR TITLE
fix(help): fix issue where hidden scripts cannot be executed

### PR DESCRIPTION
**Pull request by gunnx**
**What**:
When a script was set to be hidden from the `help` command output if you tried to run the script it would error

**Why**:
Users are unable to hide scripts and run them

**How**:

The help output function now filters scripts to check for presence of the hiddenFromHelp property

#74

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ dist
 .opt-in
 .opt-out
 npm-debug.log
+.idea/
```
Comment (kentcdodds) [inline .gitignore]: Should probably be in your global gitignore file ;-) But it's fine. I don't mind adding it.

**Comment by kentcdodds** [comment]
Oh, one thing, could you please [add yourself to the contributors](https://github.com/kentcdodds/p-s/blob/master/CONTRIBUTING.md#add-yourself-as-a-contributor)?

Then feel free to merge this yourself! (I'm adding you as a collaborator right now)

**Comment by kentcdodds** [comment]
:shipit:

File: .gitignore (hunk 1 above)
Comment (gunnx) [inline .gitignore]: Sure, I was using my work machine and we don't use global ignore

**Comment by gunnx** [comment]
I took a slightly different approach, to just filter out the scripts at the help function level

**Comment by kentcdodds** [comment]
LGTM! Go ahead and merge it yourself! Welcome to the team!

**Comment by kentcdodds** [comment]
Congratulations @gunnx! Great work :clap:

This has been automatically released :)

**Comment by gunnx** [comment]
@kentcdodds Great when will this version be available to download on npm

**Comment by kentcdodds** [comment]
Because it's automatically released when it's merged to master, it normally just takes a few minutes! See [this](http://kcd.im/write-oss) for more info on that.

**Comment by boneskull** [comment]
...it didn't seem to release.

**Comment by kentcdodds** [comment]
Shucks, this happened:

screen shot 2016-11-09 at 12 03 27 am

I'm not sure why this happened, but I'll resolve it as soon as I'm able. Thanks for letting me know.

**Comment by gunnx** [comment]
Did I miss a step?

**Comment by kentcdodds** [comment]
Definitely wasn't you. I think it was actually because I pushed a beta release out manually from my computer. I still need to figure out how to do that safely 😅

**Comment by kentcdodds** [comment]
Alrighty, this should be released and we should be good to go with automated releases now 👍

**Pull request merged**